gems/pending/openstack/openstack_handle/pagination/marker.rb

```diff
@@ -0,0 +1,25 @@
+require 'openstack_handle/pagination/base'
+
+module OpenstackHandle
+  module Pagination
+    class Marker < OpenstackHandle::Pagination::Base
+      def list
+        all_servers = call_list_method(@collection_type, @options, @method)
+        last_server = all_servers.try(:last)
+
+        while marker(last_server, @collection_type)
```

Comment:
I'd rather see this call to the `marker` method be a predicate method like `has_more_pages?` since it's used in a boolean expression.